rules/alerting.go

```diff
@@ -302,7 +321,7 @@ func (r *AlertingRule) SetEvaluationTimestamp(ts time.Time) {
 
 // GetEvaluationTimestamp returns the time the evaluation took place.
 func (r *AlertingRule) GetEvaluationTimestamp() time.Time {
-	return r.evaluationTimestamp.Load()
+	return r.evaluationTimestamp.Load().(time.Time)
```

Comment:
What about adding small helpers in `util/atomicutil` package for those type safe operations? Perhaps using generics?